src/lib/RpcServer.js

```diff
@@ -99,19 +105,21 @@ class RpcServer {
              */
             _error(message, error) {
                 this._replyTo(message, 'error',
-                    error.message ? { message: error.message, stack: error.stack, code: error.code } : { message: error } )
+                    error.message
+                        ? { message: error.message, stack: error.stack, code: error.code }
+                        : { message: error });
```

Comment:
here the object construction has been changed to inline, while before (line 52) it has been changed to one line per property. how is that possible?

Reply:
The object construction before was not _changed_ by ESLint, but was written that way by a human. ESLint just added the trailing comma.